pom.xml

```diff
@@ -126,7 +126,7 @@
         <jackson.version>2.9.8</jackson.version>
         <slf4j.version>1.7.25</slf4j.version>
         <auto-value.version>1.5.3</auto-value.version>
-        <guava.version>20.0</guava.version>
+        <guava.version>21.0</guava.version>
```

Comment:
Is that needed? I think we need to be careful to use version compatible with Beam SDK.

Reply:
lets keep this change in a separate PR.

Reply:
Addressed.